### PR TITLE
Vault: Fix issue with stale update vaults

### PR DESCRIPTION
**Pull request by coachchucksol**
Forces the vault to update from the previous state, given missed update epochs or partial updates that have not called close_vault_update_state_tracker

### CODE REVIEW

File: vault_program/src/initialize_vault_update_state_tracker.rs [hunk 1]
```diff
@@ -75,7 +75,22 @@ pub fn process_initialize_vault_update_state_tracker(
         &vault_update_state_tracker_seeds,
     )?;
 
-    let additional_assets_need_unstaking = if vault.additional_assets_need_unstaking() > 0 {
+    let needs_to_recover_from_partial_or_late_update = {
```
Comment (buffalu) [inline vault_program/src/initialize_vault_update_state_tracker.rs:78]: do you need to look at anything except for last_full_update_was_last_epoch?

Comment (coachchucksol) [reply vault_program/src/initialize_vault_update_state_tracker.rs:78]: No, it should be simplified!

File: vault_core/src/vault.rs [hunk 2]
```diff
@@ -1331,8 +1344,9 @@ mod tests {
             std::mem::size_of::<PodU16>() + // reward_fee_bps
             std::mem::size_of::<PodU16>() + // program_fee_bps
             std::mem::size_of::<PodBool>() + // is_paused
+            std::mem::size_of::<PodU64>() + // last_start_state_update_slot
             1 + // bump
-            261; // reserved
+            253; // reserved
```
Comment (buffalu) [inline vault_core/src/vault.rs:1349]: missing a field? this is 251 above

**Comment by buffalu** [comment]
what happens with the epoch rotational stuff + not enough to withdraw from non updated operators?

**Review by buffalu — APPROVED** [review]

File: vault_core/src/vault.rs (hunk 2 above)
Comment (coachchucksol) [reply vault_core/src/vault.rs:1349]: fixing

Comment (coachchucksol) [reply vault_core/src/vault.rs:1349]: next_withdrawal_fee_bps was missing ( and is missing from master branch as well )

**Review by ebatsell — APPROVED** [review]
lgtm

**Pull request merged**